universal-login-relayer/lib/core/services/DeploymentHandler.ts

```diff
@@ -19,8 +18,23 @@ class DeploymentHandler {
       state: 'Queued'
     } as Deployment;
     await this.deploymentRepository.add(deployment.hash, deployment);
-    await this.executionQueue.addDeployment(deployment);
-    return this.walletService.deploy(deployment, deviceInfo);
+    const hash = await this.executionQueue.addDeployment(deployment);
+    return hash;
+  }
+
+  async getStatus(deploymentHash: string) : Promise<DeploymentStatus | null> {
+    let deployment: Deployment;
+    try {
+      deployment = await this.deploymentRepository.get(deploymentHash);
+      const status : DeploymentStatus = {
+        deploymentHash,
+        error: deployment.error,
+        state: deployment.state,
+        transactionHash: deployment.transactionHash
+      };
+      return status;
+    } catch (_) {}
```

Comment:
`IRepository.get()` throws when item not found, so need a try-catch.

Reply:
catch only ItemNotFounds, so it does not silently fail on other exceptions